pycarwings2/pycarwings2.py

```diff
@@ -118,7 +118,12 @@ def _request(self, endpoint, params):
         else:
             params["custom_sessionid"] = ""
 
-        req = Request('POST', url=BASE_URL + endpoint, data=params).prepare()
+        req = Request(
+            'POST',
+            url=BASE_URL + endpoint,
+            data=params,
+            headers={"User-Agent": ""}
```

Comment:
we might want to leave a developers/comment/note around here explaining why we're forging the `UserAgent` to empty

Reply:
From my research, so long as it's not clearly urllib (or cURL - that took a while to debug) any useragent (or no useragent) seems to work. Setting it to "My PyCarwings Lib/2.whatever" also works, and might be considered more valid

Reply:
But setting it to default User-agent that urllib3 forces does not work.

Reply:
Yes. I was saying that rather than forging it to empty and needing a note to explain why, it could instead be set to a human-readable value that isn't rejected by the carwings servers.

Reply:
Yes, that is also something, it was promoted here: https://github.com/home-assistant/core/issues/43312#issuecomment-734779522 
Problem is that no one has tested/mapped what User-Agent headers are accepted.